src/Components/Home.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import Navbar from './Navbar'
+import GetIcons from './Icons/GetIcons'
+
+function Home() {
+  return (
+    <>
+      <Navbar
+        start={
+          <Link to="/search" aria-label="Search">
+            <GetIcons iconName="search" size={20} />
+          </Link>
+        }
+      />
+      <h2 className="text-4xl text-center">
+        LinkFree connects audiences to all of your content with just one link
```

Comment:
```suggestion
        LinkFree connects audiences to all of your content with just one link.
```

Reply:
As a header, this shouldn't have a full stop IMHO

Reply:
I originally had a `.`, but then thought it looked better without. Is it a personal preference?